packages/design-system-react/src/components/text/Text.types.ts

```diff
@@ -60,6 +60,10 @@ export type TextProps = {
    * @default TextColor.TextDefault
    */
   color?: TextColor;
+  /**
+   * Optional prop for testing purposes
+   */
+  'data-testid'?: string;
```

Comment:
Hmm unfortunately this is still needed for this issue https://github.com/MetaMask/metamask-design-system/pull/409#discussion_r1936477417